modules/@ergonode/users/src/components/Tabs/UserAvatarTab.vue

```diff
@@ -5,20 +5,91 @@
 <template>
     <CenterViewTemplate :fixed="true">
         <template #centeredContent>
-            <UserAvatarForm />
+            <UserAvatarForm
+                submit-title="SAVE CHANGES"
+                :is-submitting="isSubmitting"
+                :scope="scope"
+                :errors="errors"
+                :changed-values="changeValues"
+                @submit="onSubmit" />
         </template>
     </CenterViewTemplate>
 </template>
 
 <script>
+import {
+    ALERT_TYPE,
+} from '@Core/defaults/alerts';
+import tabFeedbackMixin from '@Core/mixins/tab/tabFeedbackMixin';
 import CenterViewTemplate from '@UI/components/Layout/Templates/CenterViewTemplate';
 import UserAvatarForm from '@Users/components/Forms/UserAvatarForm';
+import {
+    mapActions,
+    mapState,
+} from 'vuex';
 
 export default {
     name: 'UserAvatarTab',
     components: {
         UserAvatarForm,
         CenterViewTemplate,
     },
+    mixins: [
+        tabFeedbackMixin,
+    ],
+    data() {
+        return {
+            isSubmitting: false,
+        };
+    },
+    computed: {
+        ...mapState('authentication', [
+            'user',
+        ]),
+        ...mapState('user', [
+            'id',
+        ]),
+    },
+    methods: {
+        ...mapActions('authentication', [
+            'getUser',
+        ]),
+        ...mapActions('user', [
+            'updateUserAvatar',
+        ]),
+        onSubmit() {
+            if (this.isSubmitting) {
+                return;
```

Comment:
taki dziwny ten return.... nie lepiej wywalic return i zrobic !this.isSubmitting { kod }

Reply:
It's how we do our code styling